app/main.py

```diff
@@ -1,3 +1,43 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json") as file:
+        config = json.load(file)
+
+    fuel_price, customers_list, shops_list = (
+        config["FUEL_PRICE"],
+        config["customers"],
+        config["shops"]
+    )
+    customers = Customer.list_read(customers_list)
+    shops = Shop.list_read(shops_list)
+
+    for customer in customers:
```

Comment:
Seems like it is all about customer so you can move this part to customer module
But it’s not required. Just an idea or advice